lib/elixir/lib/regex.ex

```diff
@@ -412,6 +412,80 @@ defmodule Regex do
     opts
   end
 
+  @doc """
+  Returns the pattern as an embeddable string.
+
+  If the pattern was compiled with an option which cannot be represented
+  as an embeddable modifier in the current version of PCRE and strict is true
+  (the default) then an ArgumentError exception will be raised.
+
+  When strict is false the pattern will be returned as though any offending
+  options had not be used and the function will not raise any exceptions.
+
+  Embeddable modifiers/options are currently:
+    * 'i' - :caseless
+    * 'm' - :multiline
+    * 's' - :dotall, :newline, :anycrlf})
+    * 'x' - :extended
+
+  And unembeddable modifiers are
+    * 'f' - :firstline
+    * 'U' - :ungreedy
+    * 'u' - :unicode, :ucp
+
+  Any other regex compilation option not listed here is considered unembeddable.
```

Comment:
```suggestion
  Any other regex compilation option not listed here is considered unembeddable
  unless the `:strict` option is set to `false`.
```

Reply:
i see what you are getting at here, but this wording makes it sound like turning off strict makes thing embeddable. How about: "Any other regex compilation option not listed here is considered unembeddable and will raise an exception unless the `:strict` option is set to `false`".

Reply:
As you prefer, my main goal was to list the option :)

Reply:
Fix pushed as I described here.